webplugin/js/app/km-richtext-markup-1.0.js

```diff
@@ -522,6 +530,10 @@ Kommunicate.markup.getActionableFormMarkup = function(options) {
                 options.payload.splice(index,1);
             }
             else if(item.type == 'hidden' || item.type == 'radio' || item.type == 'checkbox' || item.type == 'text') {
+                options.payload[index].supported = true;
+                options.payload[index][item.type] = true;
+            } else {
+                options.payload[index].supported = false;
                 options.payload[index][item.type] = true;
             }
```

Comment:
@hardikparashar 
We can remove else if and just write the code like below:
Please check if it works.
else  {
                options.payload[index].supported = item.type == 'hidden' || item.type == 'radio' || item.type == 'checkbox' || item.type == 'text';
                options.payload[index][item.type] = true;	                
            }